pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -3247,32 +3304,6 @@ func TestFailTaskRun(t *testing.T) {
 	}
 }
 
-func Test_storeTaskSpec(t *testing.T) {
```

Comment:
TODO: equivalent unit test in the resolver reconciler package.

Reply:
These are now covered by `pkg/reconciler/resolver/taskrun_resolver_reconciler_test.go`